specs/test_formats/shuffling/README.md

```diff
@@ -20,13 +20,19 @@ count: int
 shuffled: List[int]
 ```
 
-- The `bytes32` is encoded a string, hexadecimal encoding, prefixed with `0x`.
+- The `bytes32` is encoded as a string, hexadecimal encoding, prefixed with `0x`.
 - Integers are validator indices. These are `uint64`, but realistically they are not as big.
 
-The `count` specifies the validator registry size. One should compute the shuffling for indices `0, 1, 2, 3, ..., count (exclusive)`.
-Seed is the raw shuffling seed, passed to permute-index (or optimized shuffling approach). 
+The `count` specifies the count of validators being shuffled (i.e. active validators during committee computation).
 
-## Condition
+One should test the shuffling for indices `0, 1, 2, 3, ..., count (exclusive)`.
+
+`shuffled` is a mapping from `i` to `get_shuffled_index(i)`.
+- `i` here is the index within committee-partitioned space. `i...i+N (excl.)` is used to get the validators for a committee of size `N`.
+- `get_shuffled_index(i)` here is the index within the active-validators space. Pointing to the validator assigned to the committee corresponding to `i`.
```

Comment:
I think adding type hinting to indicate the return type would be clearer.

Reply:
see b0bb3db